### ISSUE TITLE
Standardize namespace for RazorGeneratorMvcStart

### ISSUE THREAD

**Issue by rasitha1** [RazorGenerator/RazorGenerator#51]
RazorGeneratorMvcStart.cs file that gets generated should not have App_Start as part of the namespace. Rest of the files in App_Start don't have that namespace.

**Comment by davidebbo** [comment]
Fix is in 2.3.11

**Comment by vdehaven** [comment]
In 2.3.6, RazorGeneratorMvcStart.cs generated as such:
```[assembly: WebActivatorEx.PostApplicationStartMethod(typeof(E2SSPro.Web.App_Start.RazorGeneratorMvcStart), "Start")]
namespace E2SSPro.Web.App_Start { public static class RazorGeneratorMvcStart {
```

In 2.3.11, it produces:
```[assembly: WebActivatorEx.PostApplicationStartMethod(typeof(E2SSPro.Web.App_Start.RazorGeneratorMvcStart), "Start")]
namespace E2SSPro.Web { public static class RazorGeneratorMvcStart {

```
With the removal of App_Start from the namespace, the assembly attribute should also read similarly.

**Comment by davidebbo** [comment]
My bad! Fixed in 2.3.12

**Comment by vdehaven** [comment]
Works great, thank you for such a quick response!
